pulsar-functions/java-examples/src/main/java/org/apache/pulsar/functions/api/examples/typedMessageBuilderPublish.java

```diff
@@ -30,7 +32,7 @@
  * to publish to a desired topic based on config and setting various message configurations to be passed along.
  *
  */
-public class PublishFunctionWithMessageConf implements Function<String, Void> {
+public class typedMessageBuilderPublish implements Function<String, Void> {
```

Comment:
The "t" needs to be capitalized.  That is the convention for naming classes in java